content/git/concepts/tag/tag.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: 'Tag'
```

Comment:
```suggestion
Title: 'Git Tag'
```

Not to be confused by HTML tags

Reply:
This is not necessary since this is shown as a concept entry on both the URL and multiple places in the entry web page:
<img width="1451" alt="Screen Shot 2022-06-25 at 8 17 01 PM" src="https://user-images.githubusercontent.com/23144845/175794409-c49fef51-ea16-4316-9290-f6dd35528e38.png">

Reply:
@Dusch4593 Ignore what I said then